.github/workflows/build.yml

```diff
@@ -13,9 +13,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      # Install Java 8 for cross-compilation support. Setting it up before
+      # Java 11 means it comes later in $PATH (because of how setup-java works)
```

Comment:
Is there a way to ensure the order rather than relying on the implicit behavior of setup-java?

Reply:
I could set `PATH`, but the problem is that GHA installs Java to something version-specific, like `/java/jdk-11-1.11.242.8` or whatever. So it's hard to set it properly. I agree this solution isn't great, but it's only for a few months until we start compiling to Java 11 properly and then I can remove the Java 8 JDK... :confused:

Reply:
Sounds good.  Just figured I'd ask.  I'm OK with how this is for now.